src/Features/Core/Portable/xlf/FeaturesResources.fr.xlf

```diff
@@ -42,6 +42,11 @@
         <target state="translated">Les actions ne peuvent pas être vides.</target>
         <note />
       </trans-unit>
+      <trans-unit id="Add_static_modifier">
+        <source>Add the 'static' modifier</source>
+        <target state="new">Add the 'static' modifier</target>
```

Comment:
I could provide a translation for this one if you are interested.

Reply:
Feel free to add translation. Note that the loc team will still double check and update the target state once they have verified it is correct translation.